src/codegen/compile.ml

```diff
@@ -7870,7 +7870,7 @@ module MakeSerialization (Strm : Stream) = struct
       | Mut t ->
         read_alias env (Mut t) (fun get_arg_typ on_alloc ->
           let (set_result, get_result) = new_local env "result" in
-          Tagged.obj env Tagged.ObjInd [ compile_unboxed_const 0l ] ^^ set_result ^^
+          MutBox.alloc env ^^ set_result ^^
```

Comment:
Does MutBox.alloc contain a valid vanilla value on allocation? Should we worry if doesn't?